ios/.swiftlint.yml

```diff
@@ -40,6 +40,7 @@ opt_in_rules:
   - redundant_nil_coalescing
   - attributes
   - convenience_type
+  - shorthand_optional_binding
```

Comment:
Maybe a dedicated PR for this option "shorthand_optional_binding" can be better ?
I agree it makes the code more readable, that's good !

Reply:
Okay, I will revert this commit and make separate PR